src/components/Home/FAQ/MenteeFAQCollapse.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Collapse } from 'antd';
+import styles from './FAQCollapse.css'
+import {menteeFAQs} from "./FAQs";
+
+const { Panel } = Collapse;
+
+function MenteeFAQCollapse() {
+
+    return(
+        <Collapse className={styles.faqCollapseContainer}>
+            <Panel key={1} header="What are the eligibility requirements?">
+                <p className={styles.faqAnswer}>
```

Comment:
Can you use the Text component from antd?

Reply:
Sure